app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,26 @@
 </head>
 <body>
 
+  <header>
+    <nav>
+      <ul>
+        <li><%= link_to 'home', root_path%></li>
+        <li><%= link_to 'ask a question', new_question_path %></li>
+        <% if current_user %>
+        <li><%= link_to 'logout', session_path(current_user.id), method: :delete %></li>
```

Comment:
From the user's perspective, there's only one session - so make the route not need an id - either use singular resource :session rather than resources :session in the routes file.